warehouse/schema.go

```diff
@@ -22,36 +22,38 @@ type SchemaHandleT struct {
 	uploadSchema                  warehouseutils.SchemaT
 }
 
-func HandleSchemaChange(existingDataType, columnType string, columnVal interface{}) (newColumnVal interface{}, ok bool) {
+func HandleSchemaChange(existingDataType, currentDataType string, value any) (any, bool) {
```

Comment:
We should probably introduce an error in situations where conversion will lead into losing data, 
like `datetime` to `bool` 

or were conversion is invalid.

the bool, schema change can be replaced with a special error: ErrSchemaChange.

```suggestion
func HandleSchemaChange(existingDataType, currentDataType string, value any) (any, error) {
```

Reply:
Good point @lvrach